rskj-core/src/main/java/co/rsk/core/DifficultyCalculator.java

```diff
@@ -58,6 +58,11 @@ private static BlockDifficulty getBlockDifficulty(
         long curBlockTS = curBlockHeader.getTimestamp();
         int duration = constants.getDurationLimit();
         BigInteger difDivisor = constants.getDifficultyBoundDivisor();
+        if (activationConfig.isActive(ConsensusRule.RSKIP156, curBlockHeader.getNumber())
```

Comment:
This new block could be moved to a private method which returns the final divisor in order to improve legibility.